tool/gravity/cli/status.go

```diff
@@ -168,6 +171,67 @@ func statusOnce(ctx context.Context, operator ops.Operator, operationID string)
 	return status, nil
 }
 
+// statusHistory collects cluster status history and prints the information
+// to stdout.
+func statusHistory() error {
```

Comment:
Nit: TBH I would even move this to a separate module `cli/history.go` but that's optional.